recipe/meta.yaml

```diff
@@ -20,8 +20,8 @@ requirements:
     - poetry
   run:
     - python >=3.6.1
-    - allennlp >=1.1
-    - optuna >=2.2,<3
+    - allennlp >=2,<3
+    - optuna >=2.6,<3
```

Comment:
@himkt 
I reflected the changes here from https://github.com/himkt/allennlp-optuna/blob/master/pyproject.toml#L11-L14, but this means that 0.1.5 will only be usable on linux, because there are no allennlp 2.0 builds for [windows](https://github.com/conda-forge/allennlp-feedstock/pull/17) and [osx](https://github.com/conda-forge/allennlp-feedstock/pull/19).

Is >=2.0 a hard requirement?

Reply:
Hmm, the answer is yes. Optuna drops the support for AllenNLP v1, so I have to constraint AllenNLP `>=2.0.0`.